docs/source/requests.md

```diff
@@ -1723,30 +1724,42 @@ A client will receive NACK for
 Setting (enabling/disabling) a transaction author agreement for the pool.
 If transaction author agreement is set, then all write requests to Domain ledger (transactions) must include additional metadata pointing to the latest transaction author agreement's digest which is signed by the transaction author.
 
-If no transaction author agreement is set, or it's disabled, then no additional metadata is required.
+For any given version of transaction author agreement text and ratification date cannot be changed once set. Ratification date cannot be in future.
 
-Transaction author agreement can be disabled by setting an agreement with an empty text.
+If no transaction author agreement is set, or there are no active transaction author agreements, then no additional metadata is required.
 
-Each transaction author agreement has a unique version.
+Individual transaction author agreements can be disabled by setting retirement date using same transaction. Retirement date can be in future, in this case deactivation of agreement won't happen immediately, it will be automatically deactivated at required date instead. It is also possible to change (or delete) existing retirement date of agreement if it didn't occur yet.
+
+Latest transaction author agreement cannot be disabled using this transaction.  It is possible to disable all currently active transaction author agreements (including latest) using separate transaction [TRANSACTION_AUTHOR_AGREEMENT_DISABLE](#transaction_author_agreement_disable).
+
+Each transaction author agreement has a unique version. If TRANSACTION_AUTHOR_AGREEMENT transaction is sent for already existing version it is considered an update (for example of retirement date), in this case text and ratification date should be either absent or equal to original values.
 
 At least one [TRANSACTION_AUTHOR_AGREEMENT_AML](#transaction_author_agreement_aml) must be set on the ledger before submitting TRANSACTION_AUTHOR_AGREEMENT txn.
 
 - `version` (string):
 
     Unique version of the transaction author agreement
 
-
-- `text` (string):
+- `text` (string; optional):
 
     Transaction author agreement's text
 
+- `ratified` (integer; optional):
+
+    Timestamp of Transaction Author Agreement ratification date
+
+- `retired` (integer; optional):
+
+    Timestamp of Transaction Author Agreement retirement date.
+
 *Request Example*:
```

Comment:
I think it makes sense to provide two examples here: one for adding and one for update (retirement)